### DISCUSSION TITLE
Parcel did not rebuild the code when I update a local library

### DISCUSSION THREAD

**Discussion by ghost** [parcel-bundler/parcel#9183] [Q&A] [answered]
I clone a library locally to test it, but when I update the code in it, Parcel didn't rebuild the js file(which quote it by "npm link").
It looks like only the code for that library is in the project folder Parcel will watch it.
What should I do to get Parcel to watch it?

**Comment by mischnic** [comment, marked as answer]
See https://github.com/parcel-bundler/parcel/issues/4332
